src/coreclr/pal/src/arch/arm64/context2.S

```diff
@@ -212,3 +212,21 @@ LOCAL_LABEL(No_Restore_CONTEXT_CONTROL):
     ret
 
 LEAF_END RtlRestoreContext, _TEXT
+
+#ifdef __APPLE__
+
+// Incoming:
+//  x0: Context*
+//  x1: Exception*
+//
+LEAF_ENTRY RestoreCompleteContext, _TEXT
+    // We cannot restore all registers in the user mode code, so we rely on a help from kernel here. 
+    // The following instruction is an invalid instruction. In the hardware exception handler, we check
```

Comment:
Is it documented to be invalid instruction and always stay that way?

Reply:
That's a good question. I have used a code that's in a pack with other such codes used by the GC stress C.
https://github.com/dotnet/runtime/blob/8199d0a62184acfb25a8adf1d6c521770aee2bad/src/coreclr/vm/gccover.h#L101-L106
However, there seems to be a better and really guaranteed way. There is an UDF instruction with immediate 16 bit argument:
> Permanently Undefined generates an Undefined Instruction exception (ESR_ELx.EC = 0b000000). The encodings
for UDF used in this section are defined as permanently UNDEFINED.
These are instructions that have 16 high bits set to 0 and the low bits can be whatever.

I am going to move to that instead of that opcode.